src/data/mv_polynomial/cardinal.lean

```diff
@@ -89,14 +89,35 @@ calc #(mv_polynomial_fun σ R) = #R + #σ + #(ulift bool) :
 
 namespace mv_polynomial
 
+lemma cardinal_lift_mk_eq_max {σ : Type u} {R : Type v}
+  [nonempty σ] [comm_semiring R] [nontrivial R] :
+  #(mv_polynomial σ R) = max (max (cardinal.lift.{u} $ #R) $ cardinal.lift.{v} $ #σ) ℵ₀ :=
+begin
+  haveI : infinite (σ →₀ ℕ) := infinite_iff.2 ((le_max_right _ _).trans (mk_finsupp_nat σ).symm.le),
```

Comment:
#15725 provides this instance automatically.

Reply:
To clarify: I think this PR could be merged first; if that happens, @alexjbest please remember to remove this line in #15725.

Reply:
@alreadydone could you please add a comment in the code of the form `-- next line can be deleted after #15725 is merged`?

Reply:
I think it's not worth re-triggering CI by adding that line, and it's natural to remove it in #15725 because it's natural to golf away `haveI`, `letI` etc. after introducing new instances. I'll remind @alexjbest when he comes back to the PR.

Reply:
By the way, thanks for the approvals @jcommelin !